### PR TITLE
[zh-cn]: update the translation of Navigator: cookieEnabled property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/cookieenabled/index.md [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
-let cookieEnabled = navigator.cookieEnabled;
-```
+## 值
+
+一个布尔值。
+
+> **备注：** 当浏览器配置为阻止第三方的 cookies 时，如果在第三方 iframe 中调用 `navigator.cookieEnabled`，它会在 Safari、Edge Spartan 和 IE 中返回 `true`（而在此情况下尝试设置 cookie 会失败）。在 Firefox 和基于 Chromium 的浏览器中，它会返回 `false`。
 
-- `cookieEnabled` 是个[布尔值](/zh-CN/docs/Glossary/Boolean)： `true` 或 `false`。
+> **备注：** 在某些情况下，网络浏览器可能会阻止写入某些 cookies。例如，基于 Chrome 的浏览器以及一些实验版本的 Firefox 不允许创建具有 [`SameSite=None`](/zh-CN/docs/Web/HTTP/Headers/Set-Cookie#samesitesamesite-value) 属性的 cookies，除非它们通过 HTTPS 和 `Secure` 属性创建。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/navigator/cookieenabled/index.md:20]:
```suggestion
> **备注：** 在某些情况下，Web 浏览器可能会阻止写入某些 cookies。例如，基于 Chrome 的浏览器以及一些实验版本的 Firefox 不允许创建具有 [`SameSite=None`](/zh-CN/docs/Web/HTTP/Headers/Set-Cookie#samesitesamesite-value) 属性的 cookies，除非它们通过 HTTPS 创建并具有 `Secure` 属性。
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Just a nits.

File: files/zh-cn/web/api/navigator/cookieenabled/index.md [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
 slug: Web/API/Navigator/cookieEnabled
+l10n:
+  sourceCommit: ef75c1741b450c2331204be5563ee964ad5f4c48
 ---
 
-{{ ApiRef("HTML DOM") }}
+{{ApiRef("HTML DOM")}}
 
-`navigator.cookieEnabled` 返回一个布尔值，来表示当前页面是否启用了 cookie。本属性为只读属性。
+`navigator.cookieEnabled` 返回返回一个布尔值，指示是否启用了 cookies。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/cookieenabled/index.md:10]:
```suggestion
`navigator.cookieEnabled` 返回返回一个布尔值，指示是否启用了 cookie。
```

File: files/zh-cn/web/api/navigator/cookieenabled/index.md [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
+该属性是只读的。
 
-```js
-let cookieEnabled = navigator.cookieEnabled;
-```
+## 值
+
+一个布尔值。
+
+> **备注：** 当浏览器配置为阻止第三方的 cookies 时，如果在第三方 iframe 中调用 `navigator.cookieEnabled`，它会在 Safari、Edge Spartan 和 IE 中返回 `true`（而在此情况下尝试设置 cookie 会失败）。在 Firefox 和基于 Chromium 的浏览器中，它会返回 `false`。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/cookieenabled/index.md:18]:
```suggestion
> **备注：** 当浏览器配置为阻止第三方的 cookie 时，如果在第三方 iframe 中调用 `navigator.cookieEnabled`，它会在 Safari、Edge Spartan 和 IE 中返回 `true`（而在此情况下尝试设置 cookie 会失败）。在 Firefox 和基于 Chromium 的浏览器中，它会返回 `false`。
```

**Review by yin1999 — COMMENTED** [review]
复数

**Review by JasonLamv-t — APPROVED** [review]
Thank you! @T34-active  👍🏼

**Pull request merged**